### ISSUE TITLE
[BUG] menubar displays incorrectly

### ISSUE THREAD

**Issue by semvis123** [th-ch/youtube-music#292]
<img width="407" alt="image" src="https://user-images.githubusercontent.com/28219076/118647526-b8bf4700-b7e1-11eb-814d-cad1c3b309d8.png">
Sometimes this happens to the menubar, the plugin section disappears and gets replaced with `file`.
This doesn't happen every time and only seems to happen when focus has been changed to another window and after some time has passed. I don't know why this happens.

**Comment by Araxeus** [comment]
its not only the plugins section right?

it probably uses the whole default electron menu (I see "Window + Help" tabs etc)

**Comment by semvis123** [comment]
Yeah not only the plugin section disappeared, sorry for not clarifying that.
It indeed looks like the default menu.

**Comment by Araxeus** [comment]
If this happen in dev mode and you aren't getting any errors in the cmd
- This might be a native electron bug, since it should only display this menu in case of an error
